docs/proposals/multipart_trasnfer_mode.md

```diff
@@ -0,0 +1,296 @@
+* `params` - an object with additional parameters to send to the server as part of the `POST` JSON body.
+  These parameters are to be sent as provided back to the server.
+
+### `abort` action
+The `abort` action may include the `params` and `method` attributes, exactly as specified above.
+
+## Batch Request / Response Examples
+
+### Upload Batch Request
+The following is a ~10mb file upload request:
+```json
+{
+  "transfers": ["multipart", "basic"],
+  "operation": "upload",
+  "objects": [
+    {
+      "oid": "20492a4d0d84f8beb1767f6616229f85d44c2827b64bdbfb260ee12fa1109e0e",
+      "size": 10000000
+    }
+  ]
+}
+```
+
+### Upload Batch Response
+The following is a response for the same request, given an imaginary storage backend:
+
+```json
+{
+  "transfer": "multipart",
+  "objects": [
+    {
+      "oid": "20492a4d0d84f8beb1767f6616229f85d44c2827b64bdbfb260ee12fa1109e0e",
+      "size": 10000000,
+      "actions": {
+        "parts": [
+          {
+            "href": "https://foo.cloud.com/storage/upload/20492a4d0d84?part=0",
+            "header": {
+              "Authorization": "Bearer someauthorizationtokenwillbesethere"
+            },
+            "pos": 0,
+            "size": 2500000,
+            "expires_in": 86400
+          },
+          {
+            "href": "https://foo.cloud.com/storage/upload/20492a4d0d84?part=1",
+            "header": {
+              "Authorization": "Bearer someauthorizationtokenwillbesethere"
+            },
+            "pos": 2500000,
+            "size": 2500000,
+            "expires_in": 86400
+          },
+          {
+            "href": "https://foo.cloud.com/storage/upload/20492a4d0d84?part=2",
+            "header": {
+              "Authorization": "Bearer someauthorizationtokenwillbesethere"
+            },
+            "pos": 5000000,
+            "size": 2500000,
+            "expires_in": 86400
+          },
+          {
+            "href": "https://foo.cloud.com/storage/upload/20492a4d0d84?part=3",
+            "header": {
+              "Authorization": "Bearer someauthorizationtokenwillbesethere"
+            },
+            "pos": 7500000,
+            "expires_in": 86400
+          }
+        ],
+        "verify": {
+          "href": "https://lfs.mycompany.com/myorg/myrepo/multipart/verify",
+          "authenticated": true,
+          "header": {
+            "Authorization": "Basic 123abc123abc123abc123abc123="
+          },
+          "expires_in": 86400,
+          "params": {
+            "uploadId": "20492a4d0d84",
+            "partIds": [0, 1, 2, 3]
+          }
+        },
+        "abort": {
+          "href": "https://foo.cloud.com/storage/upload/20492a4d0d84",
+          "authenticated": true,
+          "header": {
+            "Authorization": "Basic 123abc123abc123abc123abc123="
+          },
+          "method": "DELETE",
+          "expires_in": 86400
+        }
```

Comment:
Can we provide example responses of what the `verify` and `abort` requests look like below?

Reply:
Added below.